src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEngine.Unix.cs

```diff
@@ -277,24 +277,9 @@ private SocketAsyncEngine()
                     throw new InternalException(err);
                 }
 
-                //
-                // Start the event loop on its own thread.
-                //
-                bool suppressFlow = !ExecutionContext.IsFlowSuppressed();
-                try
-                {
-                    if (suppressFlow) ExecutionContext.SuppressFlow();
-                    Task.Factory.StartNew(
-                        s => ((SocketAsyncEngine)s!).EventLoop(),
-                        this,
-                        CancellationToken.None,
-                        TaskCreationOptions.LongRunning,
-                        TaskScheduler.Default);
-                }
-                finally
-                {
-                    if (suppressFlow) ExecutionContext.RestoreFlow();
-                }
+                Thread thread = new Thread(s => ((SocketAsyncEngine)s!).EventLoop());
+                thread.Name = ".NET Sockets";
```

Comment:
Other threads use `.NET` as prefix, e.g. `.NET Finalizer` so I've done that too.
Other name: `.NET Events`?
`.NET SocketEvents` is too long, it gets truncated to `.NET SocketEven`.

Reply:
The name you choose seems fine.